### ISSUE TITLE
Pytorch models should accept tensors

### ISSUE THREAD

**Issue by WillEngler** [Garden-AI/garden#206]
The way we have things hooked up right now, PyTorch models only accept dataframes as input. As a user, I expected them to accept `torch.Tensor`s because the underlying model I trained accepted tensors.

MLFlow has ways to allow this. It's just that our naive first draft of model wrapping doesn't take advantage of it. Here's a blog post we can work off of to get this working: https://www.databricks.com/blog/2021/03/18/tensor-input-now-supported-in-mlflow.html

# Assumptions:
1.
2.

# Acceptance Criteria
Given I register a torch model with Garden that takes a tensor, when I remotely execute it with a tensor, then it works.
